OpenRA.Mods.Common/Traits/Buildings/TransformsIntoAircraft.cs

```diff
@@ -130,7 +130,7 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 			if (!order.Queued && activity.NextActivity != null)
 				activity.NextActivity.Cancel(self);
 
-			activity.Queue(self, new IssueOrderAfterTransform(order.OrderString, order.Target));
+			activity.Queue(new IssueOrderAfterTransform(order.OrderString, order.Target));
```

Comment:
This is in the wrong commit ("Make Tick return bool." instead of "Move ChildActivity handling into base Activity class.").